mcs/tools/corcompare/Util.cs

```diff
@@ -64,7 +75,23 @@ internal TypeDefinition GetBaseType (TypeDefinition child)
 			if (child.BaseType == null)
 				return null;
 
-			return child.BaseType.Resolve ();
+			try {
+				return child.BaseType.Resolve ();
+			} catch (AssemblyResolutionException) when (IgnoreResolutionErrors) {
+				return null;
+			}
+		}
+
+		internal MethodDefinition GetMethod (MethodReference method)
+		{
+			if (method == null)
+				throw new ArgumentNullException (nameof (method));
+
+			try {
+				return method.Resolve ();
+			} catch (AssemblyResolutionException) when (IgnoreResolutionErrors) {
+				return null;
```

Comment:
If we can't find the method, then carry on. Right now this is just used when resolving attributes.